packages/cozy-flags/src/index.js

```diff
@@ -1,17 +1,20 @@
 /* global __ENABLED_FLAGS__ */
 
-import isNode from 'detect-node'
-export { default as FlagSwitcher } from './browser/FlagSwitcher'
+const isNode = require('detect-node')
 
 const flag = isNode
   ? require('./node/flag').default
   : require('./browser/flag').default
 
+if (!isNode) {
+  flag.FlagSwitcher = require('./browser/FlagSwitcher').default
+}
+
 /**
  * Enables a list of flags
  * @param {string[]} flagsToEnable
  */
-export function enableFlags(flagsToEnable) {
+function enableFlags(flagsToEnable) {
```

Comment:
Why is the function not exported anymore ? It makes the build red : https://travis-ci.org/cozy/cozy-libs/builds/524933745#L1215

Reply:
Actually this function was not documented and is a duplicate of `flag.enable`. I searched for `enableFlags` in our codebases and it's used nowhere (which makes sense). So I updated test to use `flag.enable` and leaved this function unexported. Is it OK for you ?

Reply:
OK